### ISSUE TITLE
Add Library Installation Recommendations in Generated UI Code

### ISSUE THREAD

**Issue by Mefisto04** [memfreeme/memfree#17]
This issue addresses the lack of library installation recommendations in the generated UI code. While the code provides useful snippets, it does not indicate the necessary libraries to install for proper functionality. Implementing this feature will significantly enhance user experience by guiding users on required dependencies.

**Attached Screenshot**: 
![image](https://github.com/user-attachments/assets/0017d46e-f15d-4c99-9b4c-f777c8e8f869)

please assign me this issue so that i can start work on it.

**Comment by ahaapple** [comment]
@Mefisto04 Hi, thanks for your suggestion. How do you plan to fix this problem?

The dependencies in the code are here:
https://github.com/memfreeme/memfree/blob/main/frontend/components/code/evaluate-component.tsx

My previous plan was to provide a public github template repository, and then users can copy the code generated by memfree directly into a file and use it online. Like this one:
https://github.com/Yuyz0112/dewhale/tree/main/preview-ui

Because memfree supports ShadcnUI, ShadcnUI has a large number of components, and just telling users about the dependencies is not enough.

And tailwind also needs some configuration: https://github.com/memfreeme/memfree/blob/main/frontend/components/code/iframe-renderer.ts

I think we can add a preview top-level directory to the memfree repository, configure package.json, tailwind.config.js, and ShadcnUI components.

Do you have any good ideas and suggestions? Thanks

**Comment by Mefisto04** [comment]
Exactly as your response uses different library and component so to give package installation command at top which is used in that response will be helpful for user.

**Comment by ahaapple** [comment]
Hi, The key point is that all components or pages generated by memfree have the same dependencies, so we don't need to remind users every time we reply. We only need to tell them once in one place. A public github repository template should be the simplest. Do you have a better idea?

**Comment by ahaapple** [comment]
No, in this template project, we will define all the dependencies that may be needed, and all the Shadcn UI components will be included in it.

As long as it is the code generated by memfree. the user only need to follow the following steps to use it:
```

1. git clone

2. bun i

3. copy the code generated by memfree

3. bun dev
```

In the future, we will also consider supporting one-click deployment of UI pages to vercel or cloudflare page, which also requires a github repository template
